tfmarkt/MainWindow.xaml

```diff
@@ -7,8 +7,12 @@
         Title="TF Kalkulation" Height="600" Width="800"
         >
     <Grid>

```

Comment:
Warum hast du sachen in MainWindow geändert? War dies hier nicht für die Zusatzprodukte?